app.R

```diff
@@ -1,4 +1,359 @@
-# author: Christopher Alexander, Jennifer Hoang, Michelle Wang, Thea Wenxin
-# date: 2022-03-01
+# author: Christopher Alexander, Jennifer Hoang, Michelle Wang, Wenxin Xiang
+# date: 2022-03-15
 
-pass
+library(dash)
+library(dashHtmlComponents)
```

Comment:
```
library(dashHtmlComponents)
library(dashCoreComponents)
```

Reply:
I dont think we need dashCoreComponents as it is in dashHtmlComponents or dash . Thats what Florencia told I guess.

Reply:
I think we used these two libraries in the code.

Reply:
Well `dashBootstrapComponents` does not work for me, I cant even install it . Can you try running it without dashBootstrapComponents and let me know if it works.  If it does , lets skip .

Reply:
Ok, let's try.